include/libtorrent/extensions.hpp

```diff
@@ -158,12 +158,11 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_DISABLE_EXTENSIONS
 
 #include <vector>
-#include "libtorrent/config.hpp"
-#include "libtorrent/sha1_hash.hpp" // for sha1_hash
-#include "libtorrent/session_handle.hpp"
-#include "libtorrent/peer_connection_handle.hpp"
-#include "libtorrent/span.hpp"
-#include "libtorrent/string_view.hpp"
+
+#include <libtorrent/config.hpp>
+#include <libtorrent/span.hpp>
+#include <libtorrent/string_view.hpp>
+#include <libtorrent/socket.hpp>
```

Comment:
for some reason (probably copy&paste) it's `<>` in `dht_storage.hpp`, I will change it in both